test/test_images/helloworld-userport/helloworld.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: serving.knative.dev/v1alpha1
+kind: Configuration
```

Comment:
Can this use Service instead?

Reply:
I think `Configuration` is enough to my e2e test, copy from "helloworld_shell_test.go". I just want to do the same test about "helloworld_shell_test.go" or "TestHelloWorld" use diffrent port.